README.md

```diff
@@ -42,6 +42,10 @@ rustup target add armv7-unknown-linux-gnueabihf
 cargo build --target=armv7-unknown-linux-gnueabihf
 ```
 
+### Linux on Docker & Windows Subsystem for Linux (WSL)
```

Comment:
I think it might be better to put this note in [here](https://github.com/GuillaumeGomez/sysinfo/blob/8b21040b91c04a02c56bdba15c52abca8241ea63/src/linux/component.rs#L15-L17). What do you think?

Reply:
How would you feel about both? I'd completely skipped that doc block when trying to work out why this wasn't working for me! 🙈

Reply:
It's an acceptable solution. :smiley:

Reply:
Done. I don't know what format you like your comments in, so let me know if it's not right.